test/ref/pointwise.cpp

```diff
@@ -49,3 +52,28 @@ TEST_CASE(pointwise_test)
     std::vector<float> gold = {0, 2, 4};
     EXPECT(migraphx::verify::verify_rms_range(results_vector, gold));
 }
+
+TEST_CASE(pointwise_multi_out_test)
+{
+    migraphx::program p;
+    auto* mm = p.get_main_module();
+    migraphx::shape s{migraphx::shape::float_type, {3}};
+    auto a1  = mm->add_literal(migraphx::literal{s, {-1, 0, 1}});
+    auto a2  = mm->add_literal(migraphx::literal{s, {1, 16, 3}});
+    auto* pm = p.create_module("pointwise");
+    {
+        auto x1   = pm->add_parameter("x1", {migraphx::shape::float_type});
+        auto x2   = pm->add_parameter("x2", {migraphx::shape::float_type});
+        auto add  = pm->add_instruction(migraphx::make_op("add"), x1, x2);
+        auto sqrt = pm->add_instruction(migraphx::make_op("sqrt"), add);
+        pm->add_return({add, sqrt});
+    }
+    mm->add_instruction(migraphx::make_op("pointwise"), {a1, a2}, {pm});
+    p.compile(migraphx::make_target("ref"));
+    auto results = p.eval({}).back().get_sub_objects();
+
+    std::vector<float> gold1 = {0, 16, 4};
+    std::vector<float> gold2 = {0, 4, 2};
+    EXPECT(results[0].to_vector<float>() == gold1);
+    EXPECT(results[1].to_vector<float>() == gold2);
+}
```

Comment:
So just as a sanity check to ensure I've understood this. 

You're creating a pointwise module here, and then using it across two different inputs to get two completely different outputs and then verifying those outputs on ref.

Reply:
Yea this takes two inputs and has two outputs and we check the ref for both outputs.

Reply:
Understood. All good